replit/desktop
# Add openExternalUrl method to native API

# Why

It's often the case that we want to open up certain links/URLs in the user's default web browser instead of in the context of the native app's webview. e.g. in this case here: https://github.com/replit/repl-it-web/pull/32412#pullrequestreview-1436881651

We should just add an API method to make this easier from the web side.

# What changed

Add openExternalUrl method to native API

# Test plan 

Will use in follow up. In the meantime,
- Launch the app and open devtools
- In the console type `window.desktopAppApi.openExternalUrl("https://replit.com")`
- See URL open in external browser
